src/import-data/ImportDataDestination.ts

```diff
@@ -122,7 +174,9 @@ export const ImportDataDestination = ({
           {
             style: { marginLeft: '2rem' },
             disabled: !selectedWorkspace,
-            onClick: () => onImport(selectedWorkspace.workspace),
+            // Since button is disabled when selectedWorkspace is falsy,
+            // it can safely be asserted non-null in onClick.
```

Comment:
Would wrapping the `{}` in a ternary op enable TS to infer this? Something like
```
h(ButtonPrimary, !!selectedWorkspace
           ?  {style:{...}, disabled: true}
           :  {style:{...}, onClick: () => onImport(selectedWorkspace.workspace) });
```

Reply:
I suspect so, but I would not favor that change because:
- Other props (such as styled) have to be passed into both cases. At worst, they may get out of sync. At best, it's extra noise.
- It enforces limits on types for Button. It would make sense for `onClick` to be a required prop of Button, since a button that doesn't do anything is likely a bug. But this pattern would prevent that.